src/databricks/labs/remorph/resources/assessments/synapse/inventory/columns.sql

```diff
@@ -0,0 +1,26 @@
+--- Columns
+SELECT
+    TABLE_CATALOG
+     , TABLE_SCHEMA
+     , TABLE_NAME
+     , COLUMN_NAME
+     , ORDINAL_POSITION
+     , COLUMN_DEFAULT
+     , IS_NULLABLE
+     , DATA_TYPE
+     , CHARACTER_MAXIMUM_LENGTH
+     , CHARACTER_OCTET_LENGTH
+     , NUMERIC_PRECISION
+     , NUMERIC_PRECISION_RADIX
+     , NUMERIC_SCALE
+     , DATETIME_PRECISION
+     , CHARACTER_SET_CATALOG
+     , CHARACTER_SET_SCHEMA
+     , CHARACTER_SET_NAME
+     , COLLATION_CATALOG
+     , COLLATION_SCHEMA
+     , COLLATION_NAME
+     , DOMAIN_CATALOG
+     , DOMAIN_SCHEMA
+     , DOMAIN_NAME
+FROM INFORMATION_SCHEMA.COLUMNS ;
```

Comment:
Just a reminder that we'll need to document permissions needed to query the `SYS` and `INFORMATION_SCHEMA` and a disclaimer that no user data is actually read. We send a similar legal disclaimer out with the Snowflake profiler.